gradle.properties

```diff
@@ -1 +1 @@
-version=0.8.2-SNAPSHOT
```

Comment:
This shouldn't be bumped.  0.8.2 has yet to be released.

Reply:
Restored version